esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +214,36 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    def test_node_list_long(self):
+        """ Tests functionality of "esi node list" if using node_uuid.
+            returns the list of nodeif optional argument is not passed.
+            Test steps:
+            1) Set the environment variable using
+               export OS_FUNCTIONAL_NODE_IDENT_UUID = node_uuid
+            2) Check that node details were returned.
```

Comment:
This test actually isn't doing this, so this step should be removed. (the "offer list" and "lease list" tests check that details were returned because they have to create offers/leases, but this one doesn't since we're assuming a node already exists)

Reply:
At this point, I was trying to say that node details are in there got it what you are trying to say and removed it. Updated little bit of 3 point to make sense.